doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -169,15 +169,15 @@ The procedure to install CORTX on OVA is mentioned below.
 
     systemctl start|restart <service_name>
 
-#. Run **ip a l** and record the IP addresses of the following interfaces:
+#. Run **ip a l** and record the IP addresses of **ens32 + ens33** :
 
-   * ens32 - Management IP
-   * ens33 - Public data IP
-   * ens34 - Private data IP (if present)
-
-
-   .. image:: images/networks.png
+   For example, in the image below 
    
+   * ens32  = 192.168.16.176
+   * ens33  = 192.168.16.177 
```

Comment:
Why IP address is mentioned?

Reply:
Thanks, good question. This is just an example. It's not clear to people in the picture what are the IP addresses. Bari our intern was confused which is why I thought it might be best to list them out.

Reply:
OK but I hope we can provide a generic statement or provide a command so that the users can figure it out in their environment via "ip -a |grep ens*